src/admin-views/widgets/list.php

```diff
@@ -23,6 +23,26 @@
 }
 
 foreach ( $admin_fields as $field_id => $field ) {
+	// Handle a section start. May contain a section title.
+	// Using stripos() to allow for multiple occurrences ("section_start_1", "section_start_2" etc).
+	if ( false !== stripos( $field_id, 'section_start' ) ) {
```

Comment:
Since we want, here, to check for field ids that **start** with `section_start`, I would use `0 === stripos`.